nx/include/switch/services/capsa.h

```diff
@@ -18,31 +18,313 @@ void capsaExit(void);
+ * @param[out] width Output image width. Optional, can be NULL.
+ * @param[out] height Output image height. Optional, can be NULL.
+ * @param[in] file_id \ref CapsAlbumFileId
+ * @param[out] image RGBA8 image output buffer.
+ * @param[in] image_size Image buffer size, should be at least large enough for RGBA8 320x180.
+ * @param[out] workbuf Work buffer, cleared to 0 by the cmd before it returns.
+ * @param[in] workbuf_size Work buffer size, must be at least the size of the JPEG within the AlbumFile.
+ */
+Result capsaLoadAlbumScreenShotThumbnailImage(u64 *width, u64 *height, const CapsAlbumFileId *file_id, void* image, u64 image_size, void* workbuf, u64 workbuf_size);
+
+/**
+ * @brief Load the ScreenShotImage for the specified AlbumFile.
+ * @note Only available on [3.0.0+].
+ * @param[out] width Output image width. Optional, can be NULL.
+ * @param[out] height Output image height. Optional, can be NULL.
+ * @param[in] file_id \ref CapsAlbumFileId
+ * @param[in] opts \ref CapsScreenShotDecodeOption
+ * @param[out] image RGBA8 image output buffer.
+ * @param[in] image_size Image buffer size, should be at least large enough for RGBA8 1280x720.
+ * @param[out] workbuf Work buffer, cleared to 0 by the cmd before it returns.
+ * @param[in] workbuf_size Work buffer size, must be at least the size of the JPEG within the AlbumFile.
+ */
+Result capsaLoadAlbumScreenShotImageEx(u64 *width, u64 *height, const CapsAlbumFileId *file_id, const CapsScreenShotDecodeOption *opts, void* image, u64 image_size, void* workbuf, u64 workbuf_size);
+
+/**
+ * @brief Load the ScreenShotThumbnailImage for the specified AlbumFile.
+ * @note Only available on [3.0.0+].
+ * @param[out] width Output image width. Optional, can be NULL.
+ * @param[out] height Output image height. Optional, can be NULL.
+ * @param[in] file_id \ref CapsAlbumFileId
+ * @param[in] opts \ref CapsScreenShotDecodeOption
+ * @param[out] image RGBA8 image output buffer.
+ * @param[in] image_size Image buffer size, should be at least large enough for RGBA8 320x180.
+ * @param[out] workbuf Work buffer, cleared to 0 by the cmd before it returns.
+ * @param[in] workbuf_size Work buffer size, must be at least the size of the JPEG within the AlbumFile.
+ */
+Result capsaLoadAlbumScreenShotThumbnailImageEx(u64 *width, u64 *height, const CapsAlbumFileId *file_id, const CapsScreenShotDecodeOption *opts, void* image, u64 image_size, void* workbuf, u64 workbuf_size);
+
+/**
+ * @brief Load the ScreenShotImage for the specified AlbumFile.
+ * @note Only available on [3.0.0+].
+ * @param[out] width Output image width. Optional, can be NULL.
+ * @param[out] height Output image height. Optional, can be NULL.
+ * @param[out] attr \ref CapsScreenShotAttributeForApplication
+ * @param[in] file_id \ref CapsAlbumFileId
+ * @param[in] opts \ref CapsScreenShotDecodeOption
+ * @param[out] image RGBA8 image output buffer.
+ * @param[in] image_size Image buffer size, should be at least large enough for RGBA8 320x180.
```

Comment:
This is a ScreenShotImage, so shouldn't this be 1280x720?(same for capsaLoadAlbumScreenShotImageEx1)